src/topology/unit_test.rs

```diff
@@ -279,11 +312,15 @@ fn build_unit_test(
         })
     });
 
-    if !transform_outputs.contains_key(&definition.input.insert_at) {
-        errors.push(format!(
-            "unable to locate target transform '{}'",
-            definition.input.insert_at,
-        ));
+    for (input_target, _) in &inputs {
+        if !transform_outputs.contains_key(input_target) {
+            errors.push(format!(
+                "unable to locate target transform '{}'",
```

Comment:
It might be good usability to specify the test where this error is originating from?